src/parser/content-evaluator-module.ts

```diff
@@ -61,7 +64,7 @@ export class ContentEvaluatorModule extends BaseModule {
     const allCommentsUnClean = data.allComments || [];
     const allComments: { id: number; comment: string; author: string }[] = [];
     for (const commentObj of allCommentsUnClean) {
-      if (commentObj.user) {
+      if (commentObj.user && commentObj.user.type !== "Bot") {
```

Comment:
We should also filter out slash commands? And minimized comments?

Reply:
Yes I agree, I didn't program this part so was not sure what I should keep / remove.

Reply:
https://github.com/ubiquity-os-marketplace/text-conversation-rewards/pull/225/commits/3cbd808d2ed9c5e55a28707ea9900723534fbb60

Rfc @sshivaditya2019 I am not sure why all the comments were sent (commands, bot comments etc), now only user evaluated comments (which also are curated) are sent to evaluate the prompt which seems better, let me know if that introduces some issues on comment evaluation precision.

Reply:
It shouldn't affect it at all. I would proceed with implicit approval even if shiv doesn't reply in time for merge.